site-src/overrides/partials/copyright.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  Copyright (c) 2016-2024 Martin Donath <martin.donath@squidfunk.com>
```

Comment:
```suggestion
  Copyright (c) Kubernetes Authors and Martin Donath <martin.donath@squidfunk.com>
```

Wasn't sure how to handle this, but got some [help in Slack](https://kubernetes.slack.com/archives/C1TU9EB9S/p1717616676425929?thread_ts=1717613057.828109&cid=C1TU9EB9S) and this was the suggestion from @jberkus.